### PR TITLE
Consume Leaderboard API

### PR DESCRIPTION
**Pull request by gtekle**
## What this PR does
 - Created new game with ID `OYxzLGROB4qVtRjyvig5`.
 - Fetch (GET) scores on window load event.
 - Fetch (GET) scores on "Refresh" button click and render them.
 - POST new score on form "Submit" event to the `Leaderboard API`.

## How to test it
  -  clone the repository by running\
    `git clone https://github.com/gtekle/leaderboard.git`
  - navigate to the folder\
    `cd leaderboard`
  - Install packages\
    `npm install`
  - To run application using webpack-dev-server\
    `npm start`

## Details

- Fetch scores from `Leaderboard API` and render
- Add new score to `Leaderboard API` on form submit.
- Fetch score from `Leaderboard API` on refresh button click and render
- Fix linter errors

### CODE REVIEW

File: src/modules/AddScoreUI.js [hunk 1]
```diff
@@ -1,3 +1,20 @@
+import renderScore from './ScoreUI.js';
+import Scores from './Scores.js';
+
+const onFormSubmit = () => {
+  const addScoreForm = document.querySelector('#form-add-score');
+
+  addScoreForm.addEventListener('submit', async (event) => {
+    event.preventDefault();
```
Comment (hiromataba) [inline src/modules/AddScoreUI.js:8]: - [ ] [OPTIONAL] Please clear your `input` after submitting the form to show users that you have submitted and the form is empty so you can add another score

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi @gtekle   :wave: , 

appr


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
 
- No Linter errors :heavy_check_mark: 
- Your README file looks very professional :+1: 
- Your App is working correctly :heavy_check_mark: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/AddScoreUI.js (hunk 1 above)
Comment (gtekle) [reply src/modules/AddScoreUI.js:8]: Hi @hiromataba 
Thank you very much for the invaluable feedback and fast review.

Comment (gtekle) [reply src/modules/AddScoreUI.js:8]: Hi @hiromataba 
I have now added the reset form and set focus on name after submit functionalities.

thanks again.

**Pull request merged**